### DISCUSSION TITLE
I'm surprised this is not open-source

### DISCUSSION THREAD

**Discussion by oliveryasuna** [noctule-ide/noctule-swift-plugin#35] [General]
It would be incredible if this was open-source. There is a large community of developers who want Swift support in JetBrains IDEs, especially after the sunset of AppCode.

If you are interested in selling it, you could use a semi-OS license. That way, interested developers could still contribute.

**Comment by jansorg** [comment]
I don't do open-source development for such projects anymore.
I maintained open-source [BashSupport](https://plugins.jetbrains.com/plugin/4230-bashsupport) for a long time. It was a huge amount of work, a lot of work and one or two contributions over ~10 years.
I don't think that there's much overlap between developers writing Swift and developers knowing how to write a JetBrains IDE plugin.
